docs/source/plugins/06_solver_hooks.rst

```diff
@@ -117,7 +117,7 @@ This `hook` is used to update plugin's variables that depends on Tracer's mass f
 Source Terms
 ------------
 
-The `hooks` showed in this section can be considered as the most important. Since they allow the plugin to change the
+The `hooks` showed in this section can be considered as the most important since they allow the plugin to change the
```

Comment:
```suggestion
The `hooks` showed in this section can be considered as the most important hooks since they allow the plugin to change the
```

Reply:
I think I'll keep it as it is because using `hooks` again sounds redundant.